.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+
+_Please include a summary of the changes, relevant motivation and context_.
+
+# Issue
+
+* _Redmine Ticket URL_ or _Github Issue ID_.
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
+- [ ] My code follows the style guidelines of this project.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] New and existing tests pass locally with my changes.
```

Comment:
I think we'll want to change this one in the future since the tests should also run for PRs 🤔

Reply:
You're right, it will be changed when the ticket is created.